### DISCUSSION TITLE
How to modify font family (etc.) of rendered Markdown in notebook

### DISCUSSION THREAD

**Discussion by yuuuxt** [microsoft/vscode-jupyter#11197] [Questions and Answers] [answered]
How to modify font family (and font size, font weight if possible) of rendered Markdown contents in notebook? 

I haven't found a setting or a related issue yet.

Also, I want to modify how a notebook looks further. Is there a `css` file available for that purpose?

**Comment by rebornix** [comment, marked as answer]
We currently don't have a setting to override the font family for markdown previews in notebook. Good feature request.
